src/oom/oomd-util.c

```diff
@@ -0,0 +1,355 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/xattr.h>
+#include <unistd.h>
+
+#include "fd-util.h"
+#include "format-util.h"
+#include "oomd-util.h"
+#include "parse-util.h"
+#include "path-util.h"
+#include "procfs-util.h"
+#include "stat-util.h"
+#include "stdio-util.h"
+
+static int log_kill(pid_t pid, int sig, void *userdata) {
+        log_debug("oomd attempting to kill " PID_FMT " with signal %d", pid, sig);
```

Comment:
"with %s", signal_to_string(sig)" (signal numbers vary by architecture, and not everyone remembers all the numbers).